source/includes/release-notes/fact-release-notes-2-3.rst

```diff
@@ -0,0 +1,25 @@
+|bi| version 2.3 improves enterprise deployment support. The new
+features include:
+
+Improved Schema Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|bi-short| has been simplified to a single component.
+The schema sampling and mapping features of
+:doc:`mongodrdl </reference/mongodrdl>` have added to
+:doc:`mongosqld </reference/mongosqld>`.
+The :doc:`mongosqld </reference/mongosqld>` maps schemas itself.
+
+Kerberos Authentication Support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|bi-short| can now use `Kerberos <https://web.mit.edu/kerberos/>`_,
+an enterprise security protocol that authenticates clients through
+a trusted third server.
```

Comment:
Do we have docs on this yet? If so I'd link to that here. If not then we should follow up at some point to make sure that gets linked out.